connector/spanmetricsconnector/connector.go

```diff
@@ -520,11 +522,29 @@ func (p *connectorImp) buildAttributes(serviceName string, span ptrace.Span, res
 	if !contains(p.config.ExcludeDimensions, statusCodeKey) {
 		attr.PutStr(statusCodeKey, traceutil.StatusCodeStr(span.Status().Code()))
 	}
+
 	for _, d := range dimensions {
 		if v, ok := utilattri.GetDimensionValue(d, span.Attributes(), resourceAttrs); ok {
 			v.CopyTo(attr.PutEmpty(d.Name))
 		}
 	}
+
+	if !contains(p.config.ExcludeInstrumentationScope, instrumentationScopeNameKey) && instrumentationScope.Name() != "" {
+		attr.PutStr(instrumentationScopeNameKey, instrumentationScope.Name())
+	}
+	if !contains(p.config.ExcludeInstrumentationScope, instrumentationScopeVersionKey) && instrumentationScope.Version() != "" {
+		attr.PutStr(instrumentationScopeVersionKey, instrumentationScope.Version())
+	}
+
+	for _, p := range p.config.InstrumentationScope {
+		attrs := instrumentationScope.Attributes()
```

Comment:
so even if excluded we set those?

Reply:
Actually this can be moved to the line before the `for` statement.

> so even if excluded we set those?

Note this is a map and in the next lines we only add those that are part of the config.